src/classes/MyHandler.ts

```diff
@@ -1644,18 +1644,22 @@ export = class MyHandler extends Handler {
         if (handledByUs && offer.data('switchedState') !== offer.state) {
             if (notify) {
                 if (offer.state === TradeOfferManager.ETradeOfferState.Accepted) {
-                    this.bot.sendMessage(offer.partner, '/pre ✅ Success! The offer went through successfully.');
                     if (process.env.CUSTOM_SUCCESS_MESSAGE) {
                         this.bot.sendMessage(offer.partner, process.env.CUSTOM_SUCCESS_MESSAGE);
+                    } else {
+                        this.bot.sendMessage(
+                            offer.partner,
+                            '/pre ✅ Success! The offer has gone through successfully.'
+                        );
```

Comment:
Instead of doing things like this, just do like what I did in the past:
https://github.com/idinium96/tf2autobot/commit/92cf75d3a7af77e9d630944e521f2c35a808734b#diff-fc999ed8cfaad0491285299fe5b599cec43a5e5de745baa6d4f5e8d428d0a879L1639-L1644